docs/how-to/craft/code/example-classic-confinement-autotools.yaml

```
@@ -0,0 +1,27 @@
+# autotools-classic-example, an example of a classically-confined project with Autotools
+
+name: autotools-classic-example
+version: "1.0"
+summary: An example snap for an autotools project
+description: |
+  Shows how to create a snap for an application that uses a Makefile for
+  build and installation.
+
+grade: devel
+base: core22
+confinement: classic
+
```

I think we should add this for the autotools and make examples because they have some hardcoded references to x86-64. 

```suggestion
platforms:
  amd64:
```

---

Done: [d49dfb3](https://github.com/canonical/snapcraft/pull/5345/commits/d49dfb3e5f8e44dca56906c51ed9a8f43e9f83ac)

The repos are read-only. Shall we reactivate them or copy them? They'll eventually need switching to core24, too.

---

I vote we leave it as-is for now.  One of my proposals for next cycle is to have this code only in the docs and add spread tests for all the code used in the docs.